### ISSUE TITLE
BrokenPipeError: [Errno 32] Broken pipe

### ISSUE THREAD

**Issue by ztzthu** [time4tea/gopro-dashboard-overlay#7]
Thank you for this interesting gopro tool! However I find out I couldn't run the code when trying to draw the dashboard:

GoPro Timeseries has 9519 data points
Created 10 missing points...
FFMPEG Output is in /var/folders/30/2x18_w511fs5y00th7zd33r80000gp/T/tmpwipjm07s
Render: 0 [  0%] |                                                                                                                                                                                                                         | ETA:  --:--:--Timer(image to bytes - Called: 1, Total: 0.00799, Avg: 0.00799, Rate: 125.19)
Timer(writing to ffmpeg - Called: 0, Total: 0.00000, Avg: 0.00000, Rate: 0.00)
Timer(drawing frames - Called: 1, Total: 0.13921, Avg: 0.13921, Rate: 7.18)
Timer(program - Called: 1, Total: 5.32418, Avg: 5.32418, Rate: 0.19)
Traceback (most recent call last):
  File "venv/bin/gopro-dashboard.py", line 149, in <module>
    write_timer.time(lambda: writer.write(tobytes))
  File "/Users/******/Pictures/GoPro/venv/lib/python3.8/site-packages/gopro_overlay/timing.py", line 14, in time
    r = f()
  File "venv/bin/gopro-dashboard.py", line 149, in <lambda>
    write_timer.time(lambda: writer.write(tobytes))
BrokenPipeError: [Errno 32] Broken pipe

Environment:
I am using python 3.8.8 and macOS 11.5.2.

What are the possible issues? Thanks !

**Comment by time4tea** [comment]
Hiya. Yes, diagnostics could definitely be improved. Do you have ffmpeg installed? Take a look at the ffmpeg log (the location is printed when the program starts, so in your example it would be /var/folders/30/2x18_w511fs5y00th7zd33r80000gp/T/tmpwipjm07s ) Does that give anything useful?

**Comment by fahrsoft** [comment]
It may be worth noting that I had to go get the ffmpeg executable and install it… it’s not enough to just have the library added. I don’t know if that’s something that can be changed in your code, or at least noted in the documentation.

**Comment by ztzthu** [comment]
Hi guys, thanks for the quick reply! I found out that the version of my ffmpeg was maybe too high, I downgraded it to 3.2.4 and the problem wass solved ;)

In case of anyone interested what happend, here's the console output of the log of ffmpeg:

ffmpeg version 4.3 Copyright (c) 2000-2020 the FFmpeg developers
  built with clang version 10.0.0
  configuration: --prefix=/Users/******/opt/anaconda3 --cc=x86_64-apple-darwin13.4.0-clang --disable-doc --disable-openssl --enable-avresample --enable-gnutls --enable-hardcoded-tables --enable-libfreetype --enable-libopenh264 --enable-pic --enable-pthreads --enable-shared --disable-static --enable-version3 --enable-zlib --enable-libmp3lame
  libavutil      56. 51.100 / 56. 51.100
  libavcodec     58. 91.100 / 58. 91.100
  libavformat    58. 45.100 / 58. 45.100
  libavdevice    58. 10.100 / 58. 10.100
  libavfilter     7. 85.100 /  7. 85.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  7.100 /  5.  7.100
  libswresample   3.  7.100 /  3.  7.100
Unrecognized option 'preset'.
Error splitting the argument list: Option not found
